profiles/models/token.py

```diff
@@ -0,0 +1,63 @@
+import binascii
+import os
+
+from django.contrib.auth.models import User
+from django.core.validators import MinLengthValidator
+from django.db import models
+from django.utils import timezone
+
+
+class Token(models.Model):
```

Comment:
Why we don't use the [native Token from DRF](https://www.django-rest-framework.org/api-guide/authentication/#tokenauthentication)?